recipe/meta.yaml

```diff
@@ -61,10 +70,13 @@ requirements:
     - wheel
     - cudnn                                     # [cuda_compiler_version != "None"]
     - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
-    - libcublas-dev     # [(cuda_compiler_version or "").startswith("12")]
-    - libcusparse-dev   # [(cuda_compiler_version or "").startswith("12")]
-    - libcurand-dev     # [(cuda_compiler_version or "").startswith("12")]
-    - libcufft-dev      # [(cuda_compiler_version or "").startswith("12")]
+    {% if cuda_major >= 12 %}
+    - libcublas-dev
+    - libcusparse-dev
+    - libcurand-dev
+    - libcufft-dev
+    - cuda-cudart-dev
```

Comment:
I think this was missing and causing the issues in 

https://github.com/conda-forge/onnxruntime-feedstock/issues/93